website/i18n/ja/docusaurus-plugin-content-docs/current/guides/migrating.mdx

```diff
@@ -170,21 +170,21 @@ The format of the file is slightly different. Here is a comparison:
 
 <p align="center">
 
-| v1                 | v2               | Notes                                               |
-| ------------------ | ---------------- | --------------------------------------------------- |
-| name               | name             |                                                     |
-| description        |                  | Removed                                             |
-| author / name      | author / name    |                                                     |
-| author / email     | author / email   |                                                     |
-| version            | version          |                                                     |
-| binaryname         | outputfilename   | Changed                                             |
-| frontend / dir     |                  | Removed                                             |
-| frontend / install | frontend:install | Changed                                             |
-| frontend / build   | frontend:build   | Changed                                             |
-| frontend / bridge  |                  | Removed                                             |
-| frontend / serve   |                  | Removed                                             |
-| tags               |                  | Removed                                             |
-|                    | wailsjsdir       | The directory to generate wailsjs modules           |
+| v1                 | v2               | Notes                                                                                                                                                                      |
+| ------------------ | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
What's this enormous line? :-)

Reply:
This is the completion format generated by Crowdin and does not affect rendering.